btcjson/jsonrpc.go

```diff
@@ -9,6 +9,32 @@ import (
 	"fmt"
 )
 
+type RPCVersion string
```

Comment:
For all exported variables, constants, and types, let's write standard Go comments like:

```
// RPCVersion is a type to indicate RPC versions.
```